src/node/internal/internal_utils.ts

```diff
@@ -373,3 +373,90 @@ export function callbackify<T extends (...args: unknown[]) => Promise<unknown>>(
   // @ts-expect-error
   return callbackified;
 }
+
+export function parseEnv(content: string): Record<string, string> {
```

Comment:
This file is for regular utility functions, but this function is for "node:util". Can we seperate these 2 declarations into separate files?

Reply:
`callbackify` is a utils function that is defined in internal_utils. We need a place to put `parseEnv` that can be used by both process and utils, general utils seems like the right place? I'd be weary of adding a new file or having `node:process` directly import from `node:util`, as we might hit more cycle issues down the line.